halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/distributed/kubernetes/v1/KubernetesV1DistributedService.java

```diff
@@ -420,6 +420,7 @@ default KubernetesContainerDescription buildContainer(String name, ServiceSettin
     livenessHandlerAction.setPort(port);
     livenessHandler.setTcpSocketAction(livenessHandlerAction);
     livenessProbe.setHandler(livenessHandler);
+    livenessProbe.setInitialDelaySeconds(180);
```

Comment:
Is there a way to make this configurable? Clouddriver might take a really long time to start up if it is caching a large number of accounts or k8s cluster

Reply:
second making it configurable.

Reply:
Makes sense! Not very familiar with Halyard, but I'm guessing we'd configure this by adding a `livenessProbeInitialDelaySeconds` field to [`ServiceSettings`](https://github.com/spinnaker/halyard/blob/master/halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/ServiceSettings.java#L45)?

Reply:
One concern is that if we think that people will run into this limit and will need to override the default value, they will only find out when `clouddriver` fails to start...and then we'll be asking them to change another parameter (which is not a great experience).  I'm wondering if we should think of some way that is more general (and doesn't require more user-overridable values)...not that something comes to mind right away.

Reply:
Great point. What if we set a very high default `initialDelaySeconds`?

Reply:
Just chatted offline with @ezimanyi and we are going to revert the commit that added the `livenessProbe` until we can land on a proper configuration. We'll get some feedback from the Spinnaker k8s SIG and circle back on this.

Reply:
you could possibly go the other way for now if you wished something were in there.
Have it configurable with the default being disabled. Then you can turn it on with a value if you want it. That would satisfy short term demand if this was motivated by requests.

Reply:
That's a good option, thanks @ewiseblatt! Another option floated by @brandonnelson3 is to have a second server in each service's container and do a TCP check on that for the `livenessProbe` -- any thoughts on that? For now, I've reverted the original change until we land on the best solution.